moveit_ros/planning/kinematics_plugin_loader/src/kinematics_plugin_loader.cpp

```diff
@@ -259,6 +259,13 @@ moveit::core::SolverAllocatorFn KinematicsPluginLoader::getLoaderFunction(const
 
         std::string kinematics_solver_param_name = kinematics_param_prefix + ".kinematics_solver";
         const auto kinematics_solver = group_params_.at(known_group.name_).kinematics_solver;
+
+        if (kinematics_solver.empty())
+        {
+          RCLCPP_DEBUG(LOGGER, "No kinematics solver specified for group '%s'.", known_group.name_.c_str());
```

Comment:
What do you think about raising this log-level to 'WARN'?

Reply:
I think it's fine this way, otherwise it would show up for grippers etc. A warning will be shown if you attempt to run IK on groups without an IK configuration.